LoopKitUI/Views/Information Screens/DeliveryLimitsInformationView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  DeliveryLimitsInformationView.swift
+//  LoopKitUI
+//
+//  Created by Anna Quinlan on 7/3/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct DeliveryLimitsInformationView: View {
+    var onExit: (() -> Void)?
+    var mode: PresentationMode
+    
+    @Environment(\.presentationMode) var presentationMode
+    
+    public init(onExit: (() -> Void)?, mode: PresentationMode = .flow) {
+        self.onExit = onExit
+        self.mode = mode
+    }
+    
+    public var body: some View {
+        InformationView(
+            title: Text(LocalizedString("Delivery Limits", comment: "Title for delivery limit info page")),
+            buttonText: Text(LocalizedString("Next: Review Setting", comment: "Button to advance to setting editor")),
```

Comment:
(Doesn't need to block this PR, but since this button copy is reused across the information views, it might be appropriate to extract to reference from a single location)

Reply:
(see my recent PR...)

Reply:
Yep, updated to use the work in Rick's recent PR